### PR TITLE
document security fixes in release notes

### PR DESCRIPTION
**Pull request by rrelyea**

### CODE REVIEW

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[0a67f0b](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1359/commits/0a67f0b6be1a62db86f951a54ee0a2d0d608a57c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes/NuGet-4.3-RTM.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-msrcReleaseNotes/docs/release-notes/NuGet-4.3-RTM.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-4.3-rtm?branch=pr-en-us-1359) |
[docs/release-notes/NuGet-4.4-RTM.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-msrcReleaseNotes/docs/release-notes/NuGet-4.4-RTM.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-4.4-rtm?branch=pr-en-us-1359) |
[docs/release-notes/NuGet-4.5-RTM.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-msrcReleaseNotes/docs/release-notes/NuGet-4.5-RTM.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-4.5-rtm?branch=pr-en-us-1359) |
[docs/release-notes/NuGet-4.6-rtm.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-msrcReleaseNotes/docs/release-notes/NuGet-4.6-rtm.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-4.6-rtm?branch=pr-en-us-1359) |
[docs/release-notes/NuGet-4.7-RTM.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-msrcReleaseNotes/docs/release-notes/NuGet-4.7-RTM.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-4.7-rtm?branch=pr-en-us-1359) |
[docs/release-notes/NuGet-4.8-RTM.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-msrcReleaseNotes/docs/release-notes/NuGet-4.8-RTM.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-4.8-rtm?branch=pr-en-us-1359) |
[docs/release-notes/NuGet-4.9-RTM.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-msrcReleaseNotes/docs/release-notes/NuGet-4.9-RTM.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-4.9-rtm?branch=pr-en-us-1359) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C22%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201903222211301768-1359%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=9NPvWAzh1tV4QntpIDplRQ7CXAbG2Lw7CUNnESzSK6A%3D&st=2019-03-22T22%3A07%3A40Z&se=2019-04-22T22%3A12%3A40Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/release-notes/NuGet-4.4-RTM.md [hunk 1]
```diff
@@ -8,10 +8,17 @@ ms.topic: conceptual
...
+# NuGet 4.4 Release Notes
 
 [Visual Studio 2017 15.4 RTW](https://www.visualstudio.com/news/releasenotes/vs2017-relnotes) comes with NuGet 4.4 RTM.
 
+## Summary: What's New in 4.4.0
+
+## Summary: What's New in 4.4.3
+
+* Security Fix: Permissions on files created inside ~/.nuget are too open [#7673](https://github.com/NuGet/Home/issues/7673) [CVE-2019-0757](https://portal.msrc.microsoft.com/en-us/security-guidance/advisory/CVE-2019-0757)
+* Security Fix: Files inside of NUPKGs can have a relative path above the NUPKG directory [#7906](https://github.com/NuGet/Home/issues/7906)
```
Comment (zivkan) [inline docs/release-notes/NuGet-4.4-RTM.md]: minor: permissions on files was fixed in 4.4.2. 4.4.3 contains the files inside of nupkg fix.

File: docs/release-notes/NuGet-4.5-RTM.md [hunk 2]
```diff
@@ -7,10 +7,17 @@ ms.date: 12/4/2017
...
+# NuGet 4.5 Release Notes
 
 [Visual Studio 2017 15.5 RTW](https://www.visualstudio.com/news/releasenotes/vs2017-relnotes) comes with [NuGet 4.5 RTM](https://dist.nuget.org/win-x86-commandline/v4.5.0/nuget.exe).
 
+## Summary: What's New in 4.5.0
+
+## Summary: What's New in 4.5.3
+
+* Security Fix: Permissions on files created inside ~/.nuget are too open [#7673](https://github.com/NuGet/Home/issues/7673) [CVE-2019-0757](https://portal.msrc.microsoft.com/en-us/security-guidance/advisory/CVE-2019-0757)
+* Security Fix: Files inside of NUPKGs can have a relative path above the NUPKG directory [#7906](https://github.com/NuGet/Home/issues/7906)
```
Comment (zivkan) [inline docs/release-notes/NuGet-4.5-RTM.md]: minor: permissions on files was fixed in 4.5.2. 4.5.3 contains the files inside of nupkg fix.

File: docs/release-notes/NuGet-4.6-rtm.md [hunk 3]
```diff
@@ -7,16 +7,21 @@ ms.date: 3/7/2018
...
+## Summary: What's New in 4.6.0
 
 * We have added support for [signing packages](../create-packages/sign-a-package.md).
 * Visual Studio 2017 and nuget.exe now verifies package integrity before installing, restoring packages for [signed packages](../reference/signed-packages-reference.md).
 * We have improved performance of successive restores.
 
+## Summary: What's New in 4.6.4
+
+* Security Fix: Permissions on files created inside ~/.nuget are too open [#7673](https://github.com/NuGet/Home/issues/7673) [CVE-2019-0757](https://portal.msrc.microsoft.com/en-us/security-guidance/advisory/CVE-2019-0757)
+* Security Fix: Files inside of NUPKGs can have a relative path above the NUPKG directory [#7906](https://github.com/NuGet/Home/issues/7906)
```
Comment (zivkan) [inline docs/release-notes/NuGet-4.6-rtm.md]: minor: permissions on files was fixed in 4.6.3. 4.6.4 contains the files inside of nupkg fix.

File: docs/release-notes/NuGet-4.7-RTM.md [hunk 4]
```diff
@@ -7,16 +7,21 @@ ms.date: 5/14/2018
...
+## Summary: What's New in 4.7.0
 
 * We have augmented package signing to enable [Repository Signed packages](https://github.com/NuGet/Home/wiki/Repository-Signatures)
 
 * With Visual Studio Version 15.7, we have introduced the capability to [migrate existing projects that use the packages.config format to use PackageReference](https://docs.microsoft.com/en-us/nuget/reference/migrate-packages-config-to-package-reference) instead.
 
+## Summary: What's New in 4.7.3
+
+* Security Fix: Permissions on files created inside ~/.nuget are too open [#7673](https://github.com/NuGet/Home/issues/7673) [CVE-2019-0757](https://portal.msrc.microsoft.com/en-us/security-guidance/advisory/CVE-2019-0757)
+* Security Fix: Files inside of NUPKGs can have a relative path above the NUPKG directory [#7906](https://github.com/NuGet/Home/issues/7906)
```
Comment (zivkan) [inline docs/release-notes/NuGet-4.7-RTM.md]: minor: permissions on files was fixed in 4.7.2. 4.7.3 contains the files inside of nupkg fix.

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[03d5748](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1359/commits/03d5748ed3840d4f9b56b8a42c3ddb04ead2d367)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes/NuGet-4.3-RTM.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-msrcReleaseNotes/docs/release-notes/NuGet-4.3-RTM.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-4.3-rtm?branch=pr-en-us-1359) |
[docs/release-notes/NuGet-4.4-RTM.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-msrcReleaseNotes/docs/release-notes/NuGet-4.4-RTM.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-4.4-rtm?branch=pr-en-us-1359) |
[docs/release-notes/NuGet-4.5-RTM.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-msrcReleaseNotes/docs/release-notes/NuGet-4.5-RTM.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-4.5-rtm?branch=pr-en-us-1359) |
[docs/release-notes/NuGet-4.6-rtm.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-msrcReleaseNotes/docs/release-notes/NuGet-4.6-rtm.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-4.6-rtm?branch=pr-en-us-1359) |
[docs/release-notes/NuGet-4.7-RTM.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-msrcReleaseNotes/docs/release-notes/NuGet-4.7-RTM.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-4.7-rtm?branch=pr-en-us-1359) |
[docs/release-notes/NuGet-4.8-RTM.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-msrcReleaseNotes/docs/release-notes/NuGet-4.8-RTM.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-4.8-rtm?branch=pr-en-us-1359) |
[docs/release-notes/NuGet-4.9-RTM.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-msrcReleaseNotes/docs/release-notes/NuGet-4.9-RTM.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-4.9-rtm?branch=pr-en-us-1359) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C22%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201903222354486072-1359%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=S3LHafQIsxRTfOVKBDQs9srI%2F3OBjxBelmhkYIqeDaM%3D&st=2019-03-22T23%3A51%3A04Z&se=2019-04-22T23%3A56%3A04Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by rrelyea** [comment]
updated those 4 pages. Any other feedback.

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[fc6b6c1](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1359/commits/fc6b6c1fa7705dde21e290773bff1125292647e5)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes/NuGet-4.3-RTM.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-msrcReleaseNotes/docs/release-notes/NuGet-4.3-RTM.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-4.3-rtm?branch=pr-en-us-1359) |
[docs/release-notes/NuGet-4.4-RTM.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-msrcReleaseNotes/docs/release-notes/NuGet-4.4-RTM.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-4.4-rtm?branch=pr-en-us-1359) |
[docs/release-notes/NuGet-4.5-RTM.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-msrcReleaseNotes/docs/release-notes/NuGet-4.5-RTM.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-4.5-rtm?branch=pr-en-us-1359) |
[docs/release-notes/NuGet-4.6-rtm.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-msrcReleaseNotes/docs/release-notes/NuGet-4.6-rtm.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-4.6-rtm?branch=pr-en-us-1359) |
[docs/release-notes/NuGet-4.7-RTM.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-msrcReleaseNotes/docs/release-notes/NuGet-4.7-RTM.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-4.7-rtm?branch=pr-en-us-1359) |
[docs/release-notes/NuGet-4.8-RTM.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-msrcReleaseNotes/docs/release-notes/NuGet-4.8-RTM.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-4.8-rtm?branch=pr-en-us-1359) |
[docs/release-notes/NuGet-4.9-RTM.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-msrcReleaseNotes/docs/release-notes/NuGet-4.9-RTM.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-4.9-rtm?branch=pr-en-us-1359) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C22%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201903222355517773-1359%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=u8zDiH2MZe36Yl8B68jFwyHzM6mOTdZrbGsV5BKgA6s%3D&st=2019-03-22T23%3A52%3A23Z&se=2019-04-22T23%3A57%3A23Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by rrelyea** [comment]
@anangaur @karann-msft  - when you two are happy, please run with this change.

**Pull request merged**